components/molecules/Blocks/Blocks.js

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import displayBlock from '@/functions/displayBlock'
+
+/**
+ * Render the Blocks component.
+ *
+ * @param {array} blocks An array of blocks.
```

Comment:
This docblock needs some ❤️ 

```suggestion
 * @author WebDevStudios
 * @param {object} props            The component attributes as props.
 * @param {array} props.blocks  An array of blocks.
```